dataplane/controlplane.cpp

```diff
@@ -509,27 +507,36 @@ common::idp::getControlPlanePortStats::response cControlPlane::getControlPlanePo
 	else
 	{
 		/// all ports
-
-		for (const auto& [portId, port] : kernel_interfaces)
+		if (use_kernel_interface)
 		{
-			const auto& stats = std::get<2>(port);
-
-			uint64_t controlPlane_drops = 0;
-			for (const auto& [coreId, worker] : dataPlane->workers)
+			for (tPortId i = 0; i < portmapper.size(); ++i)
 			{
-				(void)coreId;
+				const auto& stats = kernel_stats[i];
+				const auto& portId = portmapper.ToDpdk(i);
 
-				controlPlane_drops += worker->statsPorts[portId].controlPlane_drops;
-			}
+				uint64_t controlPlane_drops = 0;
+				for (const auto& [coreId, worker] : dataPlane->workers)
+				{
+					(void)coreId;
```

Comment:
Structured bindings are not variables, they cannot be "used".
I suggest removing this line as this accomplishes nothing.
Also it's c-style cast, which is generally a good thing to avoid.

Or don't, I think it will be fixed anyway in bulk when we adopt some coding standard and set up linters

Reply:
Oh, yea, now I see. Sure, we'll modify such things later